src/bin/rav1e.rs

```diff
@@ -68,7 +69,8 @@ fn main() {
       if fi.frame_type == FrameType::KEY { ALL_REF_FRAMES_MASK } else { 1 };
     fi.intra_only = fi.frame_type == FrameType::KEY
       || fi.frame_type == FrameType::INTRA_ONLY;
-    fi.use_prev_frame_mvs = !(fi.intra_only || fi.error_resilient);
+    fi.primary_ref_frame =
+      if fi.intra_only || fi.error_resilient { PRIMARY_REF_NONE } else { (LAST_FRAME - LAST_FRAME) as u32 };
```

Comment:
Yes, it is strange indeed. There is actually a lot of code like this in libaom. It may make sense to define a separate constant for the second LAST_FRAME in the expression. If we wanted to use a different primary reference frame, we would change the first LAST_FRAME into something different, but wouldn't change the second LAST_FRAME in the expression.

Reply:
I would actually just like to a) switch to integer frame indexes and b) have LAST_FRAME = 0, but this will severely complicate porting so I only want to do it as a second pass once we have most of the contexting code ported.